DataFormats/L1THGCal/interface/HGCFETriggerDigiDefs.h

```diff
@@ -8,6 +8,8 @@
 
 #include "DataFormats/Common/interface/SortedCollection.h"
 
+#include "DataFormats/L1THGCal/interface/HGCFETriggerDigi.h"
+
 namespace l1t {
   //fwd decl. of FETriggerDigi
```

Comment:
Should this forward declaration be removed?

Reply:
@Teemperor ping

Reply:
@Teemperor ping

Reply:
You're right, updated the PR!